cmd/lifecycle/analyzer.go

```diff
@@ -224,3 +223,21 @@ func (a *analyzeCmd) platformAPIVersionGreaterThan06() bool {
 func (a *analyzeCmd) restoresLayerMetadata() bool {
 	return !a.platformAPIVersionGreaterThan06()
 }
+
+func (aa *analyzeArgs) ReadableRegistryImages() []string {
+	var readableImages []string
+	if !aa.useDaemon {
+		readableImages = append(readableImages, aa.previousImageRef, aa.runImageRef)
+	}
+	return readableImages
```

Comment:
I wonder if it's worth making a small utility function to filter out empty strings from the list returned. I see that you have [this check](https://github.com/buildpacks/lifecycle/pull/646/files#diff-c0efa96b114546b626552e11d133bf72b7e0fe50d145eb4a750ca50dfe208b2aR58) but it would be nice to not have to worry about this every time the method is invoked.

Reply:
@natalieparellano something like [this](https://play.golang.org/p/UfbbmgoPUIm)?

Reply:
That would work!

Reply:
Added in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/c76fda95682b294ec1cf7747f7d9203fa2a6f8e8)